### DISCUSSION TITLE
Error when opening postgres table

### DISCUSSION THREAD

**Discussion by gennarobotta** [dbeaver/dbeaver#8885] [Q&A] [answered]
#### System information: 
- SO: Windows 10 
- DBeaver version: 7.10
- Additional extensions: No one

#### Connection specification:
- Database name and version: Postgres 
- Driver name: PostgreSQL standard driver 42.2.5
- Do you use tunnels or proxies (SSH, SOCKS, etc)? No

#### Describe the problem you're observing:
If i open table a exception alert always popup with this description detail:
  
  java.lang.reflect.InvocationTargetException
    Internal driver error
    Internal driver error
      Errore SQL [XX000]: ERRORE: lettura del link simbolico "pg_tblspc/19345" fallita: Invalid argument
      Errore SQL [XX000]: ERRORE: lettura del link simbolico "pg_tblspc/19345" fallita: Invalid argument
        ERRORE: lettura del link simbolico "pg_tblspc/19345" fallita: Invalid argument
        ERRORE: lettura del link simbolico "pg_tblspc/19345" fallita: Invalid argument

#### Steps to reproduce, if exist:
double click on table name
#### Include any warning/errors/backtraces from the logs
Attach log file
[dbeaver.log](https://github.com/dbeaver/dbeaver/files/4724582/dbeaver.log)

**Comment by serge-rider** [comment]
Can't reproduce.

I can't find error "fallita: Invalid argument" in the log.
Could you check for debug log file, maybe there will be more details.

However in your log I see errors related to a missing dbeaver bundles.
Usually it means a corrupted dbeaver installation (it misses required bundles).
Try to reinstall dbeaver using Windows installer.

Note: when you upgrade dbeaver version using zip archive never overwrite old version files. Delete old version completely and then install new version.

**Comment by gennarobotta** [comment, marked as answer]
Thanks for the help, i was able to found the problem, was postgres that try to read the folder 19345 under the folder pg_tblspc in data, after i deleted 19345 the error was gone
